src/status_im/subs/wallet/wallet.cljs

```diff
@@ -33,6 +33,11 @@
  :<- [:wallet/ui]
  :-> :tokens-loading?)
 
+(rf/reg-sub
```

Comment:
can we have a corresponding test for new subs?

Reply:
@J-Son89 I moved `native-module/create-account-from-mnemonic` to an effect. Do we write tests for effects?

Reply:
I don't think we have so much tests for effects just yet so probably we are okay to ignore for now. 
We have contract tests now though so for any new RPC calls to Status Go we can add a contract test 👍

Reply:
@J-Son89 This is not an RPC call tho, it only calls a status-go method

Reply:
Yeah I was just mentioning that, not talking about this specific case.
I think in this pr you have to add a test for the new subs added though 👌 seems to be all that's needed

Reply:
@J-Son89 done with the tests 👍